sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapCashFlow.cs

```diff
@@ -113,14 +113,34 @@ public void ExamplePortfolioCashFlowsForCreditDefaultSwaps()
         internal static void CreateAndUpsertRecipe(string code, string scope, ModelSelection.ModelEnum model)
         {
             // CREATE a rule for reset quotes
-            var resetRule = new MarketDataKeyRule("Equity.RIC.*", "Lusid", scope, MarketDataKeyRule.QuoteTypeEnum.Price, "mid", quoteInterval: "1Y");
+            var resetRule = new MarketDataKeyRule(
+                key: "Equity.RIC.*",
+                supplier: "Lusid",
+                scope,
+                MarketDataKeyRule.QuoteTypeEnum.Price,
+                field: "mid",
+                quoteInterval: "1Y");
+            var ratesRule = new MarketDataKeyRule(
+                key: "Rates.*.*",
+                supplier: "Lusid",
+                scope,
+                MarketDataKeyRule.QuoteTypeEnum.Price,
+                field: "mid",
+                quoteInterval: "6Y");
+            var creditRule = new MarketDataKeyRule(
+                key: "Credit.*.*.*.*",
```

Comment:
This is probably fine and how should it work, but out of curiosity, why does it need so many `*` selections?

Looks like previously we had:
- `Equity.RIC.*`
- `Rates.*.*`

Why do we dig so many levels deeper for credit rules?

Reply:
Also, given this isn't exactly clear to me, I think it might be good to add a comment to explain some of the logic behind this which should benefit other end users of the examples as well. 

Maybe a top level that explains the key selection logic/rule sets.

Reply:
The credit mapping allows for a lot more granularity in the depth of the mapping rule, for fairly arbitrary reasons, at least in hindsight. 

I don't think here is the place to go into the nitty gritty on marketdata key rules. But Maybe a comment along the lines of "adding rules with long quote interval windows for cashflow tests, where we are happy to use old market data, as pricing is not a concern"